src/modules/schedule/components/calendar/calendar.scss

```diff
@@ -38,8 +47,20 @@
 }
 
 .fc {
-  .fc-timegrid-slot-label {
-    border: none;
+  // .fc-timegrid-slot-label {
+  //   border-top: none;
+  // }
+
+  // .fc-timegrid-slot-label:nth-child(odd) {
+  //   border-top: 1px solid #000;
+  // }
+
+  // .fc-timegrid-slot-minor {
+  //   border-top-style: none;
+  // }
```

Comment:
If we don't use the code remove it or add TODO if you will definetly need it in the future.